Tests/UITests/TestApp/LLMOpenAIRealtime/LLMOpenAIRealtimeTestView.swift

```diff
@@ -0,0 +1,44 @@
+//
+// This source file is part of the Stanford Spezi open source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziLLM
+import SpeziLLMOpenAI
+import SpeziLLMOpenAIRealtime
+import SwiftUI
+
+struct LLMOpenAIRealtimeTestView: View {
+    static let schema = LLMOpenAIRealtimeSchema {
+        LLMOpenAIFunctionWeather()
+    }
+    
+    @LLMSessionProvider(schema: Self.schema) var llm: LLMOpenAIRealtimeSession
+
+    @State var audio = AudioViewModel()
+    
+    var body: some View {
+        VStack {
+            LLMChatView(session: $llm)
+
+            HStack {
+                Button {
+                    Task {
+                        if audio.isRecording {
+                            audio.stop()
+                        } else {
+                            await audio.start()
+                        }
+                    }
+                } label: {
+                    Text(audio.isRecording ? "Stop Recording" : "Start Recording")
+                }.buttonStyle(.bordered)
+            }
```

Comment:
Might be nicer from a pure UI perspective if this would move into the toolbar and can use some nice symbols and color to represent state. Could even be a reusable button in the package in the future; might not be needed for a first iteration.